ansible_collections/arista/avd/plugins/plugin_utils/eos_designs_shared_utils/wan.py

```diff
@@ -371,11 +371,9 @@ def wan_flow_tracker_name(self: SharedUtils) -> str:
         """
         Return the name of the WAN flow tracking object
         Used in both network services, underlay and overlay python modules.
-
-        TODO make this configurable
-        TODO may need to return exporter name also later
         """
-        return "WAN-FLOW-TRACKER"
+        flow_tracking = get(self.hostvars, "cv_pathfinder_flow_tracking", default={})
+        return get(flow_tracking, "flow_tracker_name", default="WAN-FLOW-TRACKER")
```

Comment:
May need to change the `cv_pathfinder_flow_tracking` when you update schema
```suggestion
        return get(self.hostvars, "cv_pathfinder_flow_tracking.flow_tracker_name", default="WAN-FLOW-TRACKER")
```